src/components/WorldsConstants.js

```diff
@@ -0,0 +1,64 @@
+import * as DataObject from "./Database";
+import variables from '../styles/variables';
+
+export default WorldsConstants = {
```

Comment:
While I like the fact that the constants are all in one file, I would prefer it they are moved into Constants.js as opposed to a new Constants file

Reply:
Hmm, why not separate any constants that aren't related into their own file? Constants hierarchy shouldn't be unlike component hierarchy, no need to group them together if they're not referenced outside the worlds component.

I'm open to other points of view, that's just how I see it